apps/client/public/index.html

```diff
@@ -12,7 +12,7 @@
 
     Restricting connect-src is done at start of the client server.
     -->
-  <meta http-equiv="Content-Security-Policy" content="default-src 'self'; object-src 'none'; script-src 'self'; style-src 'self' 'unsafe-inline'; img-src 'self' https://i.scdn.co; connect-src *;" />
+  <meta http-equiv="Content-Security-Policy" content="default-src 'self'; object-src 'none'; script-src 'self'; style-src 'self' 'unsafe-inline'; img-src 'self' https://i.scdn.co; connect-src http://localhost:8080;" />
```

Comment:
Just to be sure, does this change still works when we have a custom server URL? Or does this only affects local development?

Reply:
This restricts the front end to only communicate with the API_ENDPOINT and nothing else **when** using fetch. In production mode as well as in development mode.

Reply:
@quentinguidee The startup script in the docker container replaces this in production:
https://github.com/Yooooomi/your_spotify/blob/e866e7918d2996ca5311d5d2c9d3ca64d20e7ebb/apps/client/scripts/run/variables.sh#L22-L28